core/src/main/java/org/apache/iceberg/rest/RESTSessionCatalog.java

```diff
@@ -495,7 +642,9 @@ public Catalog.TableBuilder buildTable(
   }
 
   @Override
-  public void invalidateTable(SessionContext context, TableIdentifier ident) {}
+  public void invalidateTable(SessionContext context, TableIdentifier ident) {
+    tableCache.invalidate(SessionIDTableID.of(context.sessionId(), ident));
```

Comment:
Do we have a specific practice in the Iceberg code around the `ID` or `Id` debate?
In this single line we have `SessionID` and `sessionId` 😄